tests/unit/test_wheel.py

```diff
@@ -83,7 +83,7 @@ def test_get_legacy_build_wheel_path__multiple_names(caplog):
     [
         u"pip = pip._internal.main:pip",
         u"pip:pip = pip._internal.main:pip",
-        pytest.param(u"進入點 = 套件.模組:函式", marks=skip_if_python2),
+        u"進入點 = 套件.模組:函式",
```

Comment:
The u-prefix could be dropped. That work could also be handled separately if preferred.

Reply:
Yeah, this could be done with pyupgrade.

Reply:
We'll get to that after this, I imagine.